src/javascript/ContentEditor/SelectorTypes/Text/Text.jsx

```diff
@@ -16,29 +15,22 @@ export const Text = ({field, value, id, onChange, onBlur}) => {
 
     const isPassword = field.selectorOptions?.find(option => option.name === 'password');
     const InteractiveIcon = hidePassword ? Visibility : Hidden;
```

Comment:
I guess it will be better to use a button component as we did for `Color` and `Date` input